doc/generated/ephypype.pipelines.create_pipeline_time_series_to_spectral_connectivity.rst

```diff
@@ -0,0 +1,6 @@
+ephypype.pipelines.create\_pipeline\_time\_series\_to\_spectral\_connectivity
+=============================================================================
+
+.. currentmodule:: ephypype.pipelines
+
+.. autofunction:: create_pipeline_time_series_to_spectral_connectivity
```

Comment:
This file shouldn't be committed ... it's generated automatically by sphinx gallery